oneview_redfish_toolkit/tests/test_util.py

```diff
@@ -265,20 +270,26 @@ def test_create_certs(self, oneview_client_mockup):
         self.assertTrue(os.path.exists(os.path.join("certs", "test" + ".key")))
 
     @mock.patch.object(util, 'OneViewClient')
```

Comment:
I believe some of these OneView mocks can be removed.

Reply:
Yes. Removing it.